### PR TITLE
Introduce GlyphTypeface.TryGetStream

### PR DESCRIPTION
**Pull request by Gillibald**
## What does the pull request do?

This PR introduces a new API via IGlyphTypeface2.TryGetStream to be able to get a byte representation of a loaded GlyphTypeface

## Checklist

- [ ] Added unit tests (if possible)?
- [x] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

File: src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs [hunk 1]
```diff
@@ -198,5 +200,41 @@ public bool TryGetTable(uint tag, out byte[] table)
...
+            {
+                var asset = _typeface.OpenStream();
+                var size = asset.Length;
+                var memoryBase = asset.GetMemoryBase();
+
+                if (memoryBase != IntPtr.Zero)
+                {
+                    unsafe
+                    {
+                        stream = new UnmanagedMemoryStream((byte*)memoryBase, size);
```
Comment (maxkatz6) [inline src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs:216]: Who is responsible for releasing memoryBase? Disposing UnmanagedMemoryStream won't do anything IIRC.

Comment (Gillibald) [reply src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs:216]: Skia is responsible for that we could copy the stream to some MemoryStream to avoid this

Comment (Gillibald) [reply src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs]: Okay now we always create a copy and the caller is responsible for managing the stream

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0038936-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0039576-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**